packages/core/src/lib/loaduikits.js

```diff
@@ -37,35 +37,42 @@ const readModuleFile = (kit, subPath) => {
 /**
  * Loads uikits, connecting configuration and installed modules
  * [1] Looks in node_modules for uikits.
- * [2] Only continue if uikit is enabled in patternlab-config.json
+ * [2] Map them to enabled uikit's in patternlab-config.json
  * [3] Reads files from uikit that apply to every template
  * @param {object} patternlab
  */
 module.exports = patternlab => {
   const paths = patternlab.config.paths;
 
-  const uikits = findModules(nodeModulesPath, isUIKitModule); // [1]
+  const uikitModules = findModules(nodeModulesPath, isUIKitModule); // [1]
 
-  uikits.forEach(kit => {
-    const configEntry = _.find(_.filter(patternlab.config.uikits, 'enabled'), {
-      name: `uikit-${kit.name}`,
+  _.filter(patternlab.config.uikits, 'enabled').forEach(uikit => {
+    const kit = _.find(uikitModules, {
+      name: uikit.name.replace('uikit-', ''),
```

Comment:
Could you please resolve the merge conflict at this position, resulting from dev merge?